packages/paste-core/components/minimizable-dialog/package.json

```diff
@@ -26,10 +26,11 @@
   },
   "peerDependencies": {
     "@twilio-paste/box": "^7.0.0",
+    "@twilio-paste/button": "^11.1.1",
     "@twilio-paste/design-tokens": "^8.0.0",
     "@twilio-paste/icons": "^9.0.0",
     "@twilio-paste/non-modal-dialog-primitive": "^0.3.10",
-    "@twilio-paste/popover": "^10.0.0",
+    "@twilio-paste/screen-reader-only": "^10.0.0",
```

Comment:
Note: popover isnt used in the Minimizable Dialog package, but screen reader only is.